src/pages/Dashboard/Dashboard.scss

```diff
@@ -4,3 +4,8 @@
     width: 90vw;
     margin: auto;
 }
+
+.dialogTitle {
```

Comment:
class names shouldn't be in camelCase. We are using hyphenated convention 👉  https://en.bem.info/methodology/naming-convention/ so this can be `.dialog-title`